traffic_portal/app/src/common/modules/table/deliveryServices/table.deliveryServices.tpl.html

```diff
@@ -42,38 +55,118 @@
+                    <td data-search="^{{::ds.httpBypassFqdn}}$">{{::ds.httpBypassFqdn}}</td>
+                    <td data-search="^{{::ds.id}}$">{{::ds.id}}</td>
+                    <td data-search="^{{::ds.infoUrl}}$">{{::ds.infoUrl}}</td>
+                    <td data-search="^{{::ds.initialDispersion}}$">{{::ds.initialDispersion}}</td>
+                    <td data-search="^{{::ds.ipv6RoutingEnabled}}$">{{::ds.ipv6RoutingEnabled}}</td>
                     <td data-search="^{{::getRelativeTime(ds.lastUpdated)}}$" data-order="{{::ds.lastUpdated}}">{{::getRelativeTime(ds.lastUpdated)}}</td>
-                    <td style="text-align: right;">
-                        <span ng-if="showChartsButton"><a class="link action-link" title="View Charts" ng-click="openCharts(ds, $event)"><i class="fa fa-sm fa-bar-chart"></i></a></span>
-                    </td>
+                    <td data-search="^{{::ds.longDesc}}$">{{::ds.longDesc}}</td>
+                    <td data-search="^{{::ds.longDesc1}}$">{{::ds.longDesc1}}</td>
+                    <td data-search="^{{::ds.longDesc2}}$">{{::ds.longDesc2}}</td>
+                    <td data-search="^{{::ds.maxDnsAnswers}}$">{{::ds.maxDnsAnswers}}</td>
+                    <td data-search="^{{::ds.maxOriginConnections}}$">{{::ds.maxOriginConnections}}</td>
+                    <td data-search="^{{::ds.midHeaderRewrite}}$">{{::ds.midHeaderRewrite}}</td>
+                    <td data-search="^{{::ds.multiSiteOrigin}}$">{{::ds.multiSiteOrigin}}</td>
+                    <td data-search="^{{::ds.originShield}}$">{{::ds.originShield}}</td>
+                    <td data-search="^{{::ds.orgServerFqdn}}$">{{::ds.orgServerFqdn}}</td>
+                    <td data-search="^{{::ds.profileName}}$">{{::ds.profileName}}</td>
+                    <td data-search="^{{::protocol(ds)}}$">{{::protocol(ds)}}</td>
+                    <td data-search="^{{::qstring(ds)}}$">{{::qstring(ds)}}</td>
+                    <td data-search="^{{::rrh(ds)}}$">{{::rrh(ds)}}</td>
+                    <td data-search="^{{::ds.regexRemap}}$">{{::ds.regexRemap}}</td>
+                    <td data-search="^{{::ds.regionalGeoBlocking}}$">{{::ds.regionalGeoBlocking}}</td>
+                    <td data-search="^{{::ds.remapText}}$">{{::ds.remapText}}</td>
+                    <td data-search="^{{::ds.routingName}}$">{{::ds.routingName}}</td>
+                    <td data-search="^{{::ds.signed}}$">{{::ds.signed}}</td>
+                    <td data-search="^{{::ds.signingAlgorithm}}$">{{::ds.signingAlgorithm}}</td>
+                    <td data-search="^{{::ds.tenant}}$">{{::ds.tenant}}</td>
+                    <td data-search="^{{::ds.trResponseHeaders}}$">{{::ds.trResponseHeaders}}</td>
+                    <td data-search="^{{::ds.trRequestHeaders}}$">{{::ds.trRequestHeaders}}</td>
+                    <td data-search="^{{::ds.type}}$">{{::ds.type}}</td>
+                    <td name="xmlId" data-search="^{{::ds.xmlId}}$">{{::ds.xmlId}}</td>
```

Comment:
How come this one has a `name` but other columns (and even the `xmlid` column in other templates) don't?

Reply:
it's because the existing UI test is retrieving this column by using a name selector. take this out and the test fails.